pkg/dashboard/ui/src/app/components/projects/project-table-row/project-table-row.component.js

```diff
@@ -123,9 +123,10 @@
                     return projectId;
                 })
                 .catch(function (error) {
-                    var defaultMsg = $i18next.t('functions:ERROR_MSG.DELETE_PROJECT', { lng: lng });
-
-                    return $q.reject(lodash.get(error, 'data.error', defaultMsg));
+                    var errorText = lodash.get(error, 'status') === 412                          ?
```

Comment:
Please also add a condition that the error text starts with `'Project contains'`.